src/lib/createVNode.js

```diff
@@ -1,3 +1,22 @@
+/**
+ * 가상 돔 노드 생성
+ * @param {*} type
+ * @param {*} props
+ * @param  {...any} children
+ * @returns {Object} {type, props, children}
+ */
 export function createVNode(type, props, ...children) {
-  return {};
+  return {
+    type,
+    props,
+    children: children
+      .flat(Infinity)
+      .filter(
+        (child) =>
+          child !== null &&
+          child !== undefined &&
+          child !== false &&
+          child !== true,
+      ),
```

Comment:
좀 더 깔끔하게 할 수 있는 방법이 있을 것 같다.

Reply:
저는 이런식으로 filter를 걸어주었어요!

```suggestion
      .filter((child) => value === 0 || Boolean(child)),
```